src/main/java/com/google/maps/model/EncodedPolyline.java

```diff
@@ -50,4 +50,7 @@ public String getEncodedPath() {
   public List<LatLng> decodePath() {
     return PolylineEncoding.decode(points);
   }
+
+  // Stick with the default toString; decoding the polyline to get a point count or representation
```

Comment:
I'd suggest adding a toString that includes the encoded polyline.

Reply:
Done.